pilz_robots_tutorials/pilz_tutorial_5/pilz_tutorial/launch/my_startup_application.launch

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<!-- %Tag(FULLTEXT)% -->
```

Comment:
@SansoneG What is this good for?

Reply:
I took the tags out for now and will open a separate Issue on the topic